input/input.c

```diff
@@ -1276,7 +1276,7 @@ static int parse_config(struct input_ctx *ictx, bool builtin, bstr data,
     return n_binds;
 }
 
-static int parse_config_file(struct input_ctx *ictx, char *file)
+static bool parse_config_file(struct input_ctx *ictx, char *file)
 {
     int r = 0;
```

Comment:
Might as well make r a bool while you're at it.

Reply:
`m_config_parse_config_file`'s comment says: returns: 1 on success, -1 on error, 0 if file not accessible.

Reply:
Nevermind, I was thinking of mpv.conf. Yeah I made it a bool since the function returns bool now.